src/app/store/effects/VehicleTechRecordModel.effects.spec.ts

```diff
@@ -125,43 +136,55 @@ describe('VehicleTechRecordModelEffects', () => {
 
     effects.setVinOnCreate$.subscribe(() => {
       spyOn(store, 'dispatch');
-      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith(valuePayload.vin);
-      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith(valuePayload.vrm);
-      expect(store.dispatch).toHaveBeenCalledWith(new SetVehicleTechRecordModelVinOnCreateSucess(valuePayload));
+      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith(
+        valuePayload.vin
+      );
+      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith(
+        valuePayload.vrm
+      );
+      expect(store.dispatch).toHaveBeenCalledWith(
+        new SetVehicleTechRecordModelVinOnCreateSucess(valuePayload)
+      );
     });
-
   });
 
   it('setVinOnCreate$ - should return errors for existing VIN & VRM', () => {
     const valuePayload = { vin: 'P012301230001', vrm: 'CT70002', vType: 'Trailer', error: [] };
 
     effects.setVinOnCreate$.subscribe(() => {
       spyOn(store, 'dispatch');
-      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith('P012301230001');
-      expect(store.dispatch).toHaveBeenCalledWith(new SetVehicleTechRecordModelVinOnCreateSucess(valuePayload));
+      expect(technicalRecordService.getTechnicalRecordsAllStatuses).toHaveBeenCalledWith(
+        'P012301230001'
+      );
+      expect(store.dispatch).toHaveBeenCalledWith(
+        new SetVehicleTechRecordModelVinOnCreateSucess(valuePayload)
+      );
     });
-
   });
 
   describe('getSearchResultError', () => {
     test('should return not found error', () => {
-      expect(effects.getSearchResultError({ error: 'No resources match the search criteria.' }))
-        .toBe(VEHICLE_TECH_RECORD_SEARCH_ERRORS.NOT_FOUND);
+      expect(
+        effects.getSearchResultError({ error: 'No resources match the search criteria.' })
```

Comment:
I think we need to store that in an enum somewhere. if it is coming from the backend we could have some sort of Mnemonics. For now I know we send strings for errors.